src/Agent.Plugins/PipelineCache/PipelineCacheTaskPluginBase.cs

```diff
@@ -29,6 +29,7 @@ public abstract class PipelineCacheTaskPluginBase : IAgentTaskPlugin
         protected const string ContentFormatVariableName = "AZP_CACHING_CONTENT_FORMAT";
         public Guid Id => PipelineCachePluginConstants.CacheTaskId;
         public abstract String Stage { get; }
+        public const string CalculatedFingerPrintVariableName = "CALCULATED_FINGER_PRINT";
```

Comment:
RESTORE_STEP_CALCULATED_FINGERPRINT

Also, do we use "calculate" or "resolve" or what?

Reply:
The thesaurus suggest to use "resolved"  😬  - changing to resolved, and 'resolving' the comment. :)